src/components/address/AddressSummary.js

```diff
@@ -44,39 +45,41 @@ export default function AddressSummary({ hash, received, spent, confirmed }) {
           <Row>
             <Col mobile={12} tablet>
               <Card.ItemContainer>
-                <Card.ItemLabel>Received</Card.ItemLabel>
-                <Card.ItemDetail>
-                  {hnsValues(received) || (
-                    <SkeletonWrapper>
-                      <Skeleton />
-                    </SkeletonWrapper>
-                  )}
-                </Card.ItemDetail>
+                <StackedData
+                  label="Received"
+                  value={
+                    hnsValues(received) || (
+                        <SkeletonWrapper>
+                          <Skeleton />
+                        </SkeletonWrapper>
+                    )
+                  }
+                />
```

Comment:
Let me know if we have any opinions on how I decided to layout this component when a complex value is passed to it. As you can see, I mainly just went with the "more indenting is better" approach